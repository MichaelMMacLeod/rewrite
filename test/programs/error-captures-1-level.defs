(def (capture ($x ..) ($y ..))
  (result ($x $y) ..))