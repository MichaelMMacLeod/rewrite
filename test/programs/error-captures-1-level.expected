error[E003]: variables matched under different ellipses used with same ellipsis
./test/programs/error-captures-1-level.defs:1:19
   |
 1 | (def (capture ($x ..) ($y ..))
   |                   ^^ $x matched under this ellipsis
./test/programs/error-captures-1-level.defs:1:27
   |
 1 | (def (capture ($x ..) ($y ..))
   |                           ^^ $y matched under this ellipsis
./test/programs/error-captures-1-level.defs:2:19
   |
 2 |   (result ($x $y) ..))
   |                   ^^ both used with this ellipsis
help: variables matched under different ellipses can't be used with the same ellipsis
